app/javascript/crayons/Dropdown/__stories__/Dropdown.stories.jsx

```diff
@@ -1,34 +1,34 @@
 import { h } from 'preact';
-import { withKnobs, text } from '@storybook/addon-knobs';
 import './dropdown-css-helper.scss';
 import notes from './dropdowns.md';
-import { Dropdown } from '@crayons';
+import { Dropdown, ButtonNew as Button } from '@crayons';
 
 export default {
   title: 'Components/Dropdowns',
-  decorators: [withKnobs],
-  parameters: { notes },
+  parameters: {
+    docs: {
+      page: notes,
+    },
+  },
```

Comment:
These docs can't be viewed at the moment due to an error being thrown - looks like some of the markup in the notes can't be parsed?